core/test/functional/api/v2/content/tags_spec.js

```diff
@@ -46,6 +46,17 @@ describe('Tags Content API V2', function () {
                 localUtils.API.checkResponse(jsonResponse.tags[0], 'tag', ['url']);
                 localUtils.API.checkResponse(jsonResponse.meta.pagination, 'pagination');
 
+                // Default order 'name asc' check
+                // the ordering difference is described in https://github.com/TryGhost/Ghost/issues/6104
+                // this condition should be removed once issue mentioned above ^ is resolved
+                if (process.env.NODE_ENV === 'testing-mysql') {
```

Comment:
@kirrg001 what do you think about this approach?

Reply:
It's temporary until we resolve the casing issue properly. Also clearly shows current difference between implementations

Reply:
yeah absolutely fine for now 👍